app/views/membership_applications/show.html.haml

```diff
@@ -1,5 +1,5 @@
 %header.entry-header
-  %h1.entry-title Ansökan från #{member_full_name}
+  %h1.entry-title  #{t('.title', member_full_name: member_full_name)}
```

Comment:
I found that if I use the `=`, it has to be on the next line (and indented). If I kept it on the same line, I had to use the `#{}` interpolation.